doc/api/module.md

```diff
@@ -931,6 +931,46 @@ added:
 `path` is the resolved path for the file for which a corresponding source map
 should be fetched.
 
+### `module.prepareStackTrace(error, trace)`
```

Comment:
Why is this in `node:module` though? I'd expect it to be in `util`, as it can be used in other use cases beyond modules?

Reply:
We have other source maps related methods exposed in `node:module`, like `module.findSourceMap` and `module.SourceMap`. It's my first impression that this method can be in `node:module`.

Reply:
This seems to be more about errors in general, and not limited to source maps, we also have `util.getSystemErrorName(err)` etc. in util. I can imagine users wanting to use it for other purposes (e.g. add a hook to get the structured trace without removing the default stack formatting once and for all - isn't this actually what our internal `overrideStackTrace` thing do? I can actually see us replacing `overrideStackTrace` with this internally).

Reply:
Actually we probably cannot replace `overrideStackTrace` because we still care about other people's callback. Speaking of which, can't we just expose this as the default value of `Error.prepareStackTrace`? This is ignored by V8 because we use the C++ API, so we can hang our default implementation there. Then users just do

```js
Error.prepareStackTrace = (error, trace) => {
  const defaultCallback = Error.prepareStackTrace;
  if (defaultCallback !== undefined) {
    // do what they do...
  }
  return defaultCallback(error, trace);
}
```

Reply:
Yes, it is true that exposing the default as `Error.prepareStackTrace` can be more intuitive. I've fixed up incorrect stack trace array modification in REPL and updated the PR.